### DISCUSSION TITLE
stat-computing-r-python/part-tools/04-scripts-notebooks

### DISCUSSION THREAD

**Discussion by giscus** [srvanderplas/stat-computing-r-python#28] [General]
# stat-computing-r-python/part-tools/04-scripts-notebooks

https://srvanderplas.github.io/stat-computing-r-python/part-tools/04-scripts-notebooks.html

**Comment by gauravk806** [comment]
Inside scripts.zip, there are "words.qmd" and "words.html" files but here in this webpage "scripts.qmd" and "scripts.html", file names are mentioned for them. Just leaving the note here. Thank you, Professor.
